controllers/cartController.js

```diff
@@ -1 +1,16 @@
+const { cartService } = require("../services");
 
+const postProductsInCart = async (req, res) => {
+  const { user_id, product_id, quantity } = req.body;
+
+  try {
```

Comment:
userId는 auth 미들웨어로 받아서 처리
```suggestion
  try {
    const { productId, quantity } = req.body;


```